bpf/mock/demo.sh

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
Can you rename this file? I don't think we need to check in demo code, maybe repurpose this to be a general CMock generation script?